### PR TITLE
fix(falco): update libs, fix CI for release

### PR DESCRIPTION
**Pull request by LucaGuerra**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug
/area ci

**What this PR does / why we need it**:

At some point, one day, I said "the only part of the release pipeline that we couldn't test is the final release".
We now noticed that when the release is final the `add_rpm` command will try to copy something from `/tmp/falco-rpm` into itself, failing.

Also, this updates libs to 0.11.2

This fixes it by changing the temp directory name.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by FedeDP — APPROVED** [review]
/approve

File: .github/workflows/reusable_publish_packages.yaml [hunk 1]
```diff
@@ -47,31 +47,31 @@ jobs:
         uses: actions/download-artifact@v3
         with:
           name: falco-${{ inputs.version }}-x86_64.rpm
-          path: /tmp/falco-rpm
+          path: /tmp/falco-release-rpm
```
[the remark this answers is not in the document]
Comment (FedeDP) [reply .github/workflows/reusable_publish_packages.yaml]: Thank you!

[the remark this answers is not in the document]
Comment (LucaGuerra) [reply .github/workflows/reusable_publish_packages.yaml]: I changed it to `falco-build` because the job that produces it calls it build

**Comment by FedeDP** [comment]
We will wait to bump libs to 0.11.2 too!

**Review by jasondellaluce — APPROVED** [review]
/approve

**Comment by FedeDP** [comment]
Keep this on hold to avoid stealing arm64 node with the master CI :)

**Pull request merged**